shared/testutil/block_test.go

```diff
@@ -45,7 +44,7 @@ func TestGenerateFullBlock_ThousandValidators(t *testing.T) {
 }
 
 func TestGenerateFullBlock_Passes4Epochs(t *testing.T) {
-	helpers.ClearAllCaches()
+
```

Comment:
Are these empty lines at the start of the function intentional? I think we usually don't have an empty line at the start of a function.

Reply:
I agree with you, the new lines show inconsistency. Fixed in the latest commit